gum/backend-arm64/gumstalker-arm64.c

```diff
@@ -439,6 +441,45 @@ gum_stalker_finalize (GObject * object)
   G_OBJECT_CLASS (gum_stalker_parent_class)->finalize (object);
 }
 
+static GumVirtualizationRequirements
+gum_exec_block_virtualize_linux_sysenter_insn (GumExecBlock * block,
+                                               GumGeneratorContext * gc)
+{
+  GumArm64Writer *cw = gc->code_writer;
+  gconstpointer continue_normally = cw->code + 1;
+  gconstpointer end_payload = cw->code + 3;
+  const cs_insn * insn = gc->instruction->ci;
+  const guint32 mrs_x15_nzcv = 0xd53b420f;
+  const guint32 msr_nzcv_x15 = 0xd51b420f;
+
+  gum_arm64_relocator_skip_one (gc->relocator);
+
+  if (gc->opened_prolog != GUM_PROLOG_NONE)
+    gum_exec_block_close_prolog (block, gc);
+
+  gum_arm64_writer_put_instruction (cw, mrs_x15_nzcv);
+
+  gum_arm64_writer_put_sub_reg_reg_imm (cw, ARM64_REG_X17,
+      ARM64_REG_X8, __NR_clone);
+  gum_arm64_writer_put_cbnz_reg_label (cw, ARM64_REG_X17, continue_normally);
+
+  gum_arm64_writer_put_instruction (cw, msr_nzcv_x15);
+  gum_arm64_writer_put_bytes (cw, insn->bytes, 4);
+  gum_arm64_writer_put_cbnz_reg_label (cw, ARM64_REG_X0, end_payload);
+
+  //we are on the child return to the original next instruction
```

Comment:
```suggestion
  /* We are on the child return to the original next instruction. */
```

Single-line comments should always be C-style and follow this style.

Reply:
Done